### PR TITLE
Add G.Projector

### PR DESCRIPTION
**Pull request by hidde-jan**
Add version 1.7.2 of NASA's excellent G.Projector app.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/gprojector.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'gprojector' do
+  version '1.7.2'
+  sha256 '853c350733961bc52ee73ba3f6dd7ffdcb672a94697a3b796cd97d839a498bc3'
+
+  url 'http://www.giss.nasa.gov/tools/gprojector/GProjectorOSX.dmg'
+  name 'G.Projector'
+  homepage 'http://www.giss.nasa.gov/tools/gprojector/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/gprojector.rb]: `:gratis`

**Comment by vitorgalvao** [comment]
It depends on Java, so it requires [a caveat](https://github.com/caskroom/homebrew-cask/blob/01d2c13e830a4d1f895cf7df3730952ec7fd49ac/Casks/jdownloader.rb#L13-17) (but “7+”, instead of “6+”).

**Comment by hidde-jan** [comment]
@vitorgalvao thanks!

**Pull request merged**